CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Upcoming
+* Rewrite project to build with babel instead of webpack.
```

Comment:
We choose to remove it since that has worked well for [nordnet-ui-kit](https://github.com/nordnet/nordnet-ui-kit). The reason we removed it here is the same: 

* Webpack is not really built for libraries (see eg. https://github.com/webpack/webpack/issues/5801)
* Complexity of maintaining and updating webpack
* The larger footprint that comes by building it in webpack